AppCenterAnalytics/AppCenterAnalyticsTests/MSBooleanTypedPropertyTests.m

```diff
@@ -0,0 +1,72 @@
+#import "MSBooleanTypedProperty.h"
+#import "MSConstants+Internal.h"
+#import "MSTestFrameworks.h"
+
+@interface MSBooleanTypedPropertyTests : XCTestCase
+
+@end
+
+@implementation MSBooleanTypedPropertyTests
+
+- (void)testNSCodingSerializationAndDeserializationWorks {
+
+  // If
+  MSBooleanTypedProperty *sut = [MSBooleanTypedProperty new];
```

Comment:
Why use "sut" here instead of just "property"?  It doesn't make any sense in this context to me

Reply:
I see that this is actually a pattern we use in iOS but I greatly dislike it...